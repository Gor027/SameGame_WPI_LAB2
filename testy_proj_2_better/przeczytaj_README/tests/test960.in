 LLL L \\
 L
 \ L\ \LL\ L\\\ \   \\L\  L L  L  L\L  \ LL L   LLLL \    \L \LL \\\ \ LLLLL\\
