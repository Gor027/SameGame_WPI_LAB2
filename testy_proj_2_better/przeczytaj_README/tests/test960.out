L L   \\
LLL\L\LL\L\\\\\\L\LLLL\L\LLLLLLL\\L\LL\\\\LLLLL\\
